storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -2126,7 +2126,7 @@ bool copy_back(int argc, char **argv) {
       return (false);
     }
     auto key = Encryption::get_master_key_id();
-    int ret = fscanf(f, "%lu", &key);
+    int ret = fscanf(f, "%u", &key);
```

Comment:
Ideally, this should come with a different commit, not squashed into the conflict resolution.

Reply:
it throwing compilation because of return type changed of get_master_key_id()